.github/workflows/lint.yml

```diff
@@ -35,13 +35,6 @@ jobs:
       # uses: chartboost/ruff-action@v1
       run: ruff check --output-format=github .
 
-    - name: format using black
```

Comment:
If you want to replace `black`, you need to introduce the ruff formatter like this: https://github.com/ParkenDD/parkapi-sources-v3/blob/main/.github/workflows/lint-test.yml#L33